pkg/tfgen/docs.go

```diff
@@ -1767,6 +1764,34 @@ func (g *Generator) convertHCL(e *Example, hcl, path string, languages []string)
 	isCompleteFailure := len(failedLangs) == len(languages)
 
 	if isCompleteFailure {
+		languagesByErrMsg := map[string]map[string]struct{}{}
+		for lang, convertErr := range failedLangs {
+			m := convertErr.Error()
```

Comment:
As this is used further down, can we name `m` `errMsg`?

Reply:
Good point, done.